lib/posix/Kconfig

```diff
@@ -39,6 +39,27 @@ config SEM_VALUE_MAX
 	help
 	  Maximum semaphore count in POSIX compliant Application.
 
+config PTHREAD_DYNAMIC_STACK
```

Comment:
these Kconfigs look good but does this all belong under the scope of PTHREAD_IPC? seem like it should be sibling to it.

Reply:
I think @pfalcon asked me not to use PTHREAD_IPC as a prefix. I'm easy either way.